.github/workflows/free-disk-space/action.yml

```diff
@@ -6,3 +6,18 @@ runs:
       - name: Remove /usr/local/* to free disk space
         run:  sudo rm -rf /usr/local/*
         shell: bash
+      - name: Remove .net to free disk space
+        run:  sudo apt purge dotnet*
+        shell: bash
+      - name: Remove chrome to free disk space
+        run:  sudo apt purge google-chrome-stable
+        shell: bash
+      - name: Remove temurin-jdk to free disk space
+        run:  sudo apt purge temurin*
+        shell: bash
+      - name: Remove google-cloud-sdk to free disk space
+        run:  sudo apt purge google-cloud-sdk
+        shell: bash
+      - name: Remove hhvm to free disk space
+        run:  sudo apt purge hhvm
+        shell: bash
```

Comment:
Amazing - I wonder how long we can play this game.

BTW, 'shell: bash' is needed for all those calls?

Reply:
I think we are at the end of the game. We currently need 52G and this will increase with the next feature. I set it to draft to find solution which future prove instead of purging a pill  of packges.

Reply:
None of the calls rely in bash behavior, that's why I don't see that need.